web/blueprints/facilities/__init__.py

```diff
@@ -613,10 +613,12 @@ def room_patchpanel_json(room_id: int) -> ResponseReturnValue:
 @bp.route("/room/<int:room_id>/tenancies/json")
 def room_tenancies_json(room_id: int) -> ResponseReturnValue:
     room = get_room_or_404(room_id)
+
     return TableResponse[RoomTenanciesRow](
         items=[
             RoomTenanciesRow(
-                inhabitant=user_button(tenancy.user) if tenancy.user else None,
+                inhabitant=user_button(tenancy.user) if tenancy.user is not None else None,
+                swdd_person_id=tenancy.person_id if tenancy.person_id is not None else None,
```

Comment:
Can the condition `if tenancy.user is not None else None` be shortened to `if tenancy.user else None` or does it change the meaning?
Same question for `tenancy.person_id` in line 621.

Reply:
Yes, since sqlalchemy objects are falsy iff they are `None`.